plugins/destination/clickhouse/typeconv/arrow/types/field.go

```diff
@@ -0,0 +1,95 @@
+package types
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2/lib/column"
+	"github.com/apache/arrow/go/v12/arrow"
+	"github.com/cloudquery/cloudquery/plugins/destination/clickhouse/util"
+	"github.com/cloudquery/plugin-sdk/v2/types"
+)
+
+func fieldFromColumn(col column.Interface) (*arrow.Field, error) {
+	name := util.UnquoteID(col.Name())
+	switch col := col.(type) {
+	case *column.Bool:
+		return &arrow.Field{Name: name, Type: new(arrow.BooleanType)}, nil
```

Comment:
there is no need to create those types. take a look here - https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/postgresql/client/types_pg.go#L29

Reply:
Could you elaborate?

Reply:
This is the reverse type transformation (from ClickHouse to Apache Arrow) as it's required to perform comparison in the normalized schemas.